syft/frameworks/torch/tensors/interpreters/replicated_shared.py

```diff
@@ -146,6 +249,30 @@ def get_shares_map(self):
         """
         return self.child
 
+    def __set_shares_map(self, shares_map):
+        self.child = shares_map
+        return self
+
+    def apply_to_shares(self, function, *args, **kwargs):
```

Comment:
Q: This is used only internally. Should it be ```public```?

Reply:
any reshaping operation like view, unfold, reshape (or any other non-arithmetic operation) will use this function, so it will be used externally